app/views/decidim/survey_results/survey_results/_separator_question_results.html.erb

```diff
@@ -0,0 +1,5 @@
+<div class="separator">
+  <br/>
+  <br/>
+  <br/>
```

Comment:
What do you think about do that with CSS? I think, in this case, this `<br />` is only space and we can do with styles.

https://www.quora.com/Is-br-a-web-design-best-practice

Reply:
It's temporary until we apply Joana's design

Reply:
ok!